lifechanger/src/app/header/header.component.css

```diff
@@ -0,0 +1,85 @@
+.header {
+  height: 6.2em;
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 100;
+  & .header__nav {
+    display: flex;
+    justify-content: space-between;
+    position: fixed;
+    background-color: var(--white);
+    top: 2.6em;
+    left: 20.8em;
+    width: 76rem;
+    /* height: 59px; */
+    height: 2em;
+    & .header__logo {
+      height: 5.4em;
+      width: 18em;
+      & .header__logo-image {
+        width: 100%;
+        height: 100%;
+      }
+    }
+    & .header__nav-container {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      width: 52em;
+      margin-right: 12px;
+      & .header__nav-links {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        width: 40em;
+        /* height: 26px; */
```

Comment:
same here

Reply:
Thank you, dear Mentor.
I deleted the unnecessary comments, please, check once again:)